main.scss

```diff
@@ -1 +1,192 @@
 @import "node_modules/govuk-frontend/all";
```

Comment:
Yes, that is correct. I will update to pick only the needed ones.

Reply:
Since this code is not yet published to npm I'm not going to worry about these styles which are currently being used for previewing the component.
They will be vendored on the initial consumption without including the import line.